CLoses#1

I have updated Validations.py file

